appveyor.yml

```diff
@@ -4,17 +4,24 @@
 environment:
   matrix:
     - nodejs_version: "0.10"
+      npm_version: "1.4.4"
     - nodejs_version: "0.12"
+      npm_version: "1.4.4"
     - nodejs_version: "4"
+      npm_version: "latest"
     - nodejs_version: "6"
+      npm_version: "latest"
     - nodejs_version: "7"
+      npm_version: "latest"
+    - nodejs_version: "8"
+      npm_version: "latest"
 
 # Install scripts. (runs after repo cloning)
 install:
   # Get the latest stable version of Node
   - ps: Install-Product node $env:nodejs_version
   # Typical npm stuff.
-  - IF %nodejs_version% LSS 1 npm -g install npm
+  - IF %nodejs_version% LSS 1 npm -g install npm@%npm_version%
```

Comment:
I think we can just hardcode "1.4.4" here and forgo the `npm_version` additions since the 'npm install' on this line is only executed for node versions < 1.0.0.